heron/proto/scheduler.proto

```diff
@@ -28,6 +29,11 @@ message RestartTopologyRequest {
   required int32 container_index = 2;
 }
 
+message UpdateTopologyRequest {
+  required heron.proto.system.PackingPlan currentPackingPlan = 1;
+  required heron.proto.system.PackingPlan proposedPackingPlan = 2;
+}
+
```

Comment:
Just wondering if this is a request send by Topology Master to Executor or something else?